### ISSUE TITLE
Install DLC on Jetson Nano

### ISSUE THREAD

**Issue by Leeyt4541** [DeepLabCut/DeepLabCut#2417]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hi all,

I worked so long to try to install DLC in Jetson Nano but i failed to find a successful way to install it. Here are what I tried:

1. I tried to install Anaconda but Jetson Nano doesn’t support Anaconda but Archiconda. I installed Archiconda.
After running “conda env create -f DEEPLABCUT.yaml”, the error:
Executing transaction: failed.
ERROR conda.core.link:_execute(507): An error occurred while installing package ‘conda-forge:attrs-22.1.0-pyh71513ae_1’.
FileNotFoundError(2, “No such file or directory: ‘/home/omni/archiconda3/envs/DEEPLABCUT/bin/python3.9’”)

2. The docker cannot be installed in Jeston Nano.
After running “deeplabcut-docker bash”, the error:
failed to solve: process "/bin/sh -c mkdir -p /home’ did not complete successfully: exit code 1, buid failed and container stopped.

3. I tried ‘pip install deeplabcut"[gui,tf]"’, but
“error: failed building wheel for wxpython. Could not build wheels for wxpython, which is required to install pyproject-based projects.”

4. I also tried to build the custom conda env, which use:
conda create -n DLC python=3.8
and
pip install 'deeplabcut[gui,tf]'
The error: legacy-install-failure
Encountered error while trying to install package.
wxpython
note: This is an issue with the package mentioned above, not pip.
hint: see above for output from the failure.

So basically I tried all methods to install DLC but all failed. And I didn’t find anyone who tried to build the DLC on Jetson Nano. Could someone help me install that useful toolbox in Jetson Nano? Thanks in advance!

Best,
Yuntao

### Operating System

Jetson Nano Ubuntu18.04

### DeepLabCut version

My installation is unsuccessful and I would like to install the newest version.

### DeepLabCut mode

single animal

### Device type

GPU: 128-core Maxwell
CPU: Quad-core ARM A57 @ 1.43 GHz

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @Leeyt4541, it should not try to install `wxPython`, as it is no longer a dependency for the GUI.

Could you try the following:
```bash
conda create -n DLC python=3.9
conda activate DLC
pip install "deeplabcut[gui,tf]==2.3.7"
```

**Comment by MMathisLab** [comment]
also there is NO way to install dlc on a jetson. We only support using DLC-lIve on a Jetson, please see: 

https://github.com/DeepLabCut/DeepLabCut-live

and related paper: https://elifesciences.org/articles/61909
